engine-helpers/src/index.ts

```diff
@@ -303,6 +303,18 @@ export interface SetupForImagesetOptions {
   background?: Imageset;
 }
 
+/** Options for [CaptureFrame] */
+export interface CaptureFrameOptions {
+  /** The desired image width. */
+  width: number;
+
+  /** The desired image height. */
+  height: number;
+
+  /** The desired image format (e.g. `"image/jpeg"`). */
```

Comment:
We should explicitly specify that the value of this parameter should be a MIME type, if that's indeed the case. With string-valued parameters it's always good to specify the desired format of the string in as much detail as possible.

(By the same token, we should specify that the width and height are in pixels, here.)

Reply:
Good call! I've updated the descriptions for these fields.

Reply:
Did you see my other comment about the Cranko updates?

Reply:
I somehow missed that before! Tags have been updated